docs/transformations/ExpressionUnification.md

```diff
@@ -0,0 +1,32 @@
+# Expression Unification
+
+## Overview
+
+This transformation unifies binary expressions with commutative integer operands.
+
+## Restrictions
+
+* operands are only integer
+* operators are only ["*", "+", "|", "&", "^"]
+
+## Examples
+* 
```

Comment:
Why do you use * symbol here? you can delete them and use line separator between examples or add something like this: First example, second example